server/command.go

```diff
@@ -47,7 +48,7 @@ func (p *Plugin) getCommandResponse(responseType, text string) *model.CommandRes
 		ResponseType: responseType,
 		Text:         text,
 		Username:     GITLAB_USERNAME,
-		IconURL:      p.getConfiguration().ProfileImageURL,
+		IconURL:      path.Join(*p.API.GetConfig().ServiceSettings.SiteURL, "plugins", manifest.Id, "assets", "profile.png"),
```

Comment:
You can use [`GetBundlePath`](https://developers.mattermost.com/extend/plugins/server/reference/) instead.

Reply:
Nop, GetBundlePath return an absolute path e.g. `/home/user/....` I want an http URL. Maybe a cool helper could be `GetHTTPBundlePath`!?

Reply:
:man_facepalming: Your correct.

Reply:
What would `GetHTTPBundlePath()` return? `ServiceSettings.SiteURL` + "plugins" + `pluginID`?

Reply:
Yes :100: